.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 test/tmp
+/
```

Comment:
Wait what? That doesn't seem right?

Have you two reviewed the diff after finishing it? That'd be very useful, so I don't have to catch all the typos.